### PR TITLE
Allow multiple font variant values to be used, fixes #175

### LINKED ISSUE #175: Support multiple font-variants
The following:
```jsx
styled.view`
 font-variant: small-caps oldstyle-nums;
`
```

 should map to
```jsx
<View style={{fontVariant: ['small-caps', 'oldstyle-nums']}}
```

It looks like instead it only chooses one of the font-variants provided.

### PR DESCRIPTION
**Pull request by finnp**
Fixes #175

cc @jacobp100 ☺️

### CODE REVIEW

File: src/transforms/index.js [hunk 1]
```diff
@@ -39,9 +40,18 @@ const margin = directionFactory({
   prefix: 'margin',
 })
 const padding = directionFactory({ prefix: 'padding' })
-const fontVariant = tokenStream => ({
-  fontVariant: [tokenStream.expect(IDENT)],
-})
+const fontVariant = tokenStream => {
```
Comment (jacobp100) [inline src/transforms/index.js:43]: Sorry to be nitpicky - can we put this in a separate file? This file was only really for very small definitions

**Review by jacobp100 — APPROVED** [review]

File: src/transforms/index.js (hunk 1 above)
Comment (finnp) [reply src/transforms/index.js:43]: Yep, I agree I'll change this!

**Comment by finnp** [comment]
@jacobp100 Just moved it to another file. Do you want to have another look?

**Comment by jacobp100** [comment]
Thank you! I'll try and do a release ASAP

**Comment by jacobp100** [comment]
Sorry I forgot to say - this was released. Either 3.1.0 or 3.2.0 which I just released now

**Pull request merged**